src/Sulu/Bundle/AdminBundle/Resources/js/components/CKEditor5/ckeditor5.scss

```diff
@@ -1,60 +1,79 @@
 @import '../../containers/Application/colors.scss';
 
-$textEditorBorderRadius: 3px;
-$textEditorButtonBorderRadius: 2px;
-$textEditorToolbarBackgroundColor: $silver;
-$textEditorBackgroundColor: $white;
-$textEditorBorderColor: $silver;
-$textEditorButtonActiveBackground: $dustyGray;
-
 :global {
-    .ck.ck-toolbar {
-        background-color: $textEditorToolbarBackgroundColor;
-        border-radius: $textEditorBorderRadius;
+    :root {
+        /* font size */
+        --ck-font-size-base: 12px;
+        --ck-icon-size: 12px;
+
+        /* shadows */
+        --ck-color-shadow-inner: none;
+        --ck-color-shadow-drop: none;
+        --ck-color-button-default-active-shadow: none;
+        --ck-color-button-on-active-shadow: none;
+
+        /* toolbar */
+        --ck-ui-component-min-height: 20px;
+        --ck-color-toolbar-background: $silver;
```

Comment:
I would still create variables like you see at the top of the old version of this file. We have decided to do this in order to let people override these styles later. This also applies to all other color references in this file.